server/Dockerfile

```diff
@@ -16,8 +16,16 @@ RUN apt update \
 
 # install data/make dependencies
 RUN npm install -g mapshaper
+RUN apt update && apt -y install jq
 
 # install python dependencies
 RUN pip install --upgrade pip
 COPY ./requirements.txt /usr/local/aclu/elections-api/requirements.txt
-RUN pip install -r requirements.txt
\ No newline at end of file
+RUN pip install -r requirements.txt
+COPY ./dev-requirements.txt /usr/local/aclu/elections-api/dev-requirements.txt
+RUN pip install -r dev-requirements.txt
+
+# smartcrop + dependencies
+RUN apt update && apt -y install libopencv-dev
+RUN npm install -g opencv --unsafe-perm=true
```

Comment:
Similarly, you could merge the npm installs into one line (and possibly the pip installs too)

Reply:
yeah totally, merged both!